typescript/infra/src/config/helloworld-getters.ts

```diff
@@ -0,0 +1,37 @@
+import { ChainMap, MultiProvider, RouterConfig } from '@hyperlane-xyz/sdk';
+import { objMap } from '@hyperlane-xyz/utils';
+
+import { Contexts } from '../../config/contexts';
+import { routingIsm } from '../../config/routingIsm';
+import { getRouterConfig } from '../../scripts/utils';
+
+import { DeployEnvironment } from './environment';
+import { HelloWorldConfig } from './helloworld';
+
+export const helloWorldConfig = (
```

Comment:
Do these belong with https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/config/environments/testnet3/helloworld.ts ?

Reply:
I actually had meant to rm this function - the other one is useful for mainnet2 as well as testnet3. But i moved `helloWorldRouterConfig` into `helloworld.ts` instead of this file